app/assets/stylesheets/components/_option-select.scss

```diff
@@ -107,10 +107,17 @@
 
 .app-c-option-select__filter {
   display: none;
+  padding-top: govuk-spacing(2);
+  margin-bottom: govuk-spacing(2);
+}
+
+.app-c-option-select__filter-input {
+  padding-left: 33px;
```

Comment:
Does this need to be 33px? Can it be 30px instead, and therefore use a govuk-spacing variable?

Reply:
Yeah, I didn't like that either. I'm trying to line up the text in the input with the text of the labels beneath - it looks neater to me.